doc/tutorials/klondike/step5.md

```diff
@@ -413,9 +413,41 @@ FlameGame. The new world contains (almost) everything we need to play the game a
 re-created during each of the above actions.
 
 
-### A stripped-down KlondikeGame class
+### KlondikeWorld Class
 
-Here is the new code for the KlondikeGame class (what is left of it).
+
+#### Introduction to `World` in Flame
+
+In Flame, `World` is a special type of `Component` that can contain other `Components`,
+such as Piles in the Klondike game. Think of `World` as the playground where everything 
+happens—dealing cards, arranging them, and player actions.
+
+You can learn more about `World` in game programming here:
+
+- An introduction to `World` in game programming: [Ecampus](https://ecampusontario.pressbooks.pub/gamedesigndevelopmenttextbook/chapter/what-is-a-game-world/)
```

Comment:
We shouldn't have links to external resourced in here, you can link to the world section in the docs instead.